client/src/main/java/org/apache/celeborn/client/ShuffleClientImpl.java

```diff
@@ -729,7 +729,6 @@ Map<Integer, Integer> reviveBatch(
             pushExcludedWorkers.remove(loc.getPeer().hostAndPushPort());
           }
         } else if (StatusCode.STAGE_ENDED.getValue() == statusCode) {
-          stageEnded(shuffleId);
```

Comment:
Better to call `stageEndShuffleSet.put(shuffleId)` ?

Reply:
Yea